### ISSUE TITLE
How to manipulate or create a new loss function ??

### ISSUE THREAD

**Issue by shivom9713** [junyanz/pytorch-CycleGAN-and-pix2pix#1194]
Dear sir , 
              I am working on Infrared to visual image conversion using GANs , unfortunately cycle consistency loss and adversarial loss is not enough to make good conversions even after good amount of training on a sufficiently large dataset .
Can you help me what loss functions should i use and how to make changes in loss function code ??
Thanks

**Comment by seawee1** [comment]
Hi,
there are many follow-up works based on CycleGAN which might be better suited to tackle your problem, probably also the semi-supervised setting. I can very much recommend this repository to look up some alternative methods: [Awesome Image Translation](https://github.com/weihaox/awesome-image-translation)

**Comment by shivom9713** [comment]
Thanks a lot sebastian !!
